fixture/different.transformPath/src/a

```diff
@@ -0,0 +1 @@
+a
```

Comment:
Which empty lines do you mean? I do not see any empty lines in the fixture files